src/components/metadata/metadata-row.js

```diff
@@ -21,7 +21,7 @@ const MetaDataRow = ({
   children,
 }) => {
   const showList = Array.isArray(value);
-  const showObject = typeof value === 'object' && !showList;
+  const showObject = typeof value === 'object' && value !== null && !showList;
```

Comment:
Thanks for this implementation!! I might put the null value check to the check in the return statement ( line 26 of the same file) so that way the label would not get rendered as well for null values

Reply:
Actually this is for backward-compatibility. Current behaviour in production is that if it's an empty value, e.g. `null`, it gets displayed as `-`. If we want to change that behaviour, we need to get Gab to okay first. So let's keep it backward compat for now.